electron_app/src/electron-main.js

```diff
@@ -323,13 +324,14 @@ app.on('ready', () => {
             preload: preloadScript,
             nodeIntegration: false,
             sandbox: true,
-            enableRemoteModule: false,
+            enableRemoteModule: true,
             // We don't use this: it's useful for the preload script to
             // share a context with the main page so we can give select
             // objects to the main page. The sandbox option isolates the
             // main page from the background script.
             contextIsolation: false,
             webgl: false,
+            nativeWindowOpen: true,
```

Comment:
note: this option is experimental
See https://electronjs.org/docs/api/browser-window

Reply:
I'll test feature without it

Reply:
ok it works without it. Sorry for commiting it, I was inspired by Rocket Chat implementation, so I copied their settings ;)